### PR TITLE
Task8

### PR DESCRIPTION
**Pull request by Mateen-Ahmad5**
- Implement media queries to adjust the layout, font sizes, and spacing for different screen sizes. 
- Ensure that the Portfolio page is visually appealing and user-friendly on both desktop and mobile devices. 
- There’s no hard requirement for the layout. But it should be easy to browse and understand the portfolio on a mobile device.

### CODE REVIEW

File: data-view.css [hunk 1]
```diff
@@ -1,48 +1,57 @@
 /* styles.css */
 body {
```
Comment (akbar-ashraf) [inline data-view.css:2]: use Boxing Sizing for better Responsive experince

File: editstyle.css [hunk 2]
```diff
@@ -262,9 +264,139 @@ nav ul li a:hover {
...
-} 
+}
 .close-btn:hover {
   background-color: #d32f2f;
 }
 
-  
\ No newline at end of file
+/* Responsive Styles */
+
```
Comment (akbar-ashraf) [inline editstyle.css]: Use Media queries from Bootstrap or tailwind for better Responsiveness breakpoints

**Review by akbar-ashraf — CHANGES_REQUESTED** [review]
please reolve these points

File: data-view.css (hunk 1 above)
Comment (Mateen-Ahmad5) [reply data-view.css:2]: fix

File: editstyle.css (hunk 2 above)
Comment (Mateen-Ahmad5) [reply editstyle.css]: ok

**Review by akbar-ashraf — APPROVED** [review]

**Pull request merged**